phpcs.xml

```diff
@@ -4,6 +4,7 @@
     <rule ref="PSR12">
         <exclude name="Squiz.Functions.FunctionDeclarationArgumentSpacing.SpaceAfterEquals" />
         <exclude name="Squiz.Functions.FunctionDeclarationArgumentSpacing.SpaceBeforeEquals" />
+        <exclude name="Generic.WhiteSpace.ScopeIndent.IncorrectExact" />
```

Comment:
@tbg-jasonfung how about an XML comment explaining the temp change here and cross referencing the issue we'll use to track reversing it when the bug is fixed?

Reply:
Sounds good, updated now